config/jobs/kubernetes/sig-node/dra.jinja

```diff
@@ -18,7 +18,7 @@ presubmits:
 {%- set testgrid_dashboards = testgrid_dashboards + ", sig-node-containerd" %}
 {%- set runtime = "containerd" %}
 {%- endif %}
-{%- if ci and not all_features %}
+{%- if ci and (release_informing == "true") %}
```

Comment:
This should better be set explicitly instead of tying it indirectly to other attributes of the job.